src/Scripting/CSharpTest/CommandLineRunnerTests.cs

```diff
@@ -82,6 +82,35 @@ public void Await()
         }
 
         [Fact]
+        public void DisplayResults()
+        {
+            var runner = CreateRunner(input:
+@"using static System.Globalization.CultureInfo;
+DefaultThreadCurrentCulture = GetCultureInfo(""en-GB"")
+Math.PI
+DefaultThreadCurrentCulture = GetCultureInfo(""de-DE"")
+Math.PI
+");
+            runner.RunInteractive();
+
+            AssertEx.AssertEqualToleratingWhitespaceDifferences(
+$@"Microsoft (R) Visual C# Interactive Compiler version {CompilerVersion}
+Copyright (C) Microsoft Corporation. All rights reserved.
+
+Type ""#help"" for more information.
+> using static System.Globalization.CultureInfo;
+> DefaultThreadCurrentCulture = GetCultureInfo(""en-GB"")
+[en-GB]
+> Math.PI
+3.1415926535897931
+> DefaultThreadCurrentCulture = GetCultureInfo(""de-DE"")
+[de-DE]
+> Math.PI
+3,1415926535897931
+>", runner.Console.Out.ToString());
+        }
+
```

Comment:
I'd like to see additional tests here;

1) What if I set the culture via Thread.CurrentCulture = new CultureInfo("de-DE"), is this respected?
2) What if I fire off a background thread, does the culture correctly flow onto it, for example, `await Task.Run(() => { return 10.ToString() };`.